Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -658,6 +682,38 @@ WarpX::ComputeEightWaysExtensions() {
 #endif
 }
 
+void
+WarpX::ApplyBCKCorrection(const int idim) {
+#if defined(AMREX_USE_EB) and !defined(WARPX_DIM_RZ)
+    auto const &cell_size = CellSize(maxLevel());
+
+    const amrex::Real dx = cell_size[0];
+    const amrex::Real dy = cell_size[1];
+    const amrex::Real dz = cell_size[2];
+
+    for (amrex::MFIter mfi(*Bfield_fp[maxLevel()][idim]); mfi.isValid(); ++mfi) {
+
+        amrex::Box const &box = mfi.validbox();
```

Comment:
Do you know if tiling on CPU would be possible here? I'm talking about adding `amrex::TilingIfNotGPU()` as a second argument when you define `mfi` and also replacing `mfi.validbox()` with `mfi.tilebox()`. I don't know if the algorithm would be compatible with tiling. If so, there could be a speedup on CPU (not sure if this could matter or if it would be negligible anyways). See the AMReX documentation [here](https://amrex-codes.github.io/amrex/docs_html/Basics.html#mfiter-and-tiling) for more info on `MFIter` with/without tiling.

Reply:
Thanks a lot for the explanation. The kernel is completely local so tiling should work here. As you said we probably don't save much time, but it's still better to have it for consistency with other parts of the code.